templates/CRM/Admin/Page/MailSettings.hlp

```diff
@@ -1,3 +1,4 @@
+@@ -0,0 +1,38 @@
```

Comment:
I like the help bubble title (grin).

Reply:
But more importantly, there's a mismatch for activity type and source since one is in the form and the other is in the page, so the bubbles don't work since it can't find the hlp file. (This is obvious if you have popups turned off since you get a red error box just opening the form.)

Reply:
Yes, thought I'd done this one, but I must have lost it somehow. The other help texts didn't work either, now they do.

Reply:
Now the help bubbles don't work for the add new form. This area appears to be broken some time after 5.63 but before this PR: https://lab.civicrm.org/dev/core/-/issues/4480